core/kazoo_stdlib/src/kz_network_utils.erl

```diff
@@ -320,8 +320,15 @@ verify_cidr(IP, CIDR) when is_binary(IP) ->
 verify_cidr(IP, CIDR) when is_binary(CIDR) ->
     verify_cidr(IP, kz_term:to_list(CIDR));
 verify_cidr(IP, CIDR) ->
-    Block = inet_cidr:parse(CIDR),
-    case inet:parse_address(IP) of
+    Block = try inet_cidr:parse(CIDR)
```

Comment:
What about something like:

```erlang
verify_cidr(IP, CIDR) ->
    try inet_cidr:parse(CIDR) of
        Block -> verify_block(IP, Block)
    catch
        'error':'invalid_cidr' -> 'false'
    end.

verify_block(IP, Block) ->
    case inet:parse_address(IP) of
        {'ok', IPTuple} -> inet_cidr:contains(Block, IPTuple);
        {'error', _} -> 'false'
    end.
```

The try block followed by case block seems clunky to me.